src/middlewared/middlewared/plugins/system_dataset/hierarchy.py

```diff
@@ -209,8 +208,8 @@ def get_system_dataset_spec(pool_name: str, uuid: str) -> list:
                 'snapdir': 'hidden',
             },
             'chown_config': {
-                'uid': WEBSHARE_UID,
-                'gid': WEBSHARE_GID,
+                'uid': 0,
+                'gid': 0,
```

Comment:
Hmm... was this a security issue with original PR? Will system dataset setup catch this change and correct ownership?